dataland-frontend/tests/e2e/fixtures/lksg/LksgDataFixtures.ts

```diff
@@ -50,19 +65,61 @@ export function generateProductionSite(undefinedProbability = 0.5): LksgProducti
  * @returns 0 to 5 random production sites
  */
 export function generateArrayOfProductionSites(undefinedProbability = 0.5): LksgProductionSite[] {
-  return Array.from({ length: faker.number.int({ min: 0, max: 5 }) }, () =>
-    generateProductionSite(undefinedProbability)
+  return generateArray(() => generateProductionSite(undefinedProbability));
+}
+
+/**
+ * Generates a random product
+ * @returns a random product
+ */
+function generateProduct(): LksgProduct {
+  return {
+    productName: faker.commerce.productName(),
+    productionSteps: valueOrUndefined(generateArray(() => faker.commerce.productName())),
+    relatedCorporateSupplyChain: valueOrUndefined(faker.commerce.productName()),
+  };
+}
+
+/**
+ * Generates a random procurement category
+ * @returns random procurement category
+ */
+function generateProcurementCategory(): LksgProcurementCategory {
+  const numberOfSuppliersPerCountryCodeAsMap = new Map<string, number>(
+    generateArray(() => [generateIso2CountryCode(), valueOrNull(faker.number.int({ min: 0, max: 50 }))!])
   );
+  return {
+    procuredProductTypesAndServicesNaceCodes: generateArray(() => faker.commerce.productName()),
```

Comment:
These should be random nace codes.   So I suggest we just pick some nace codes from the NaceCode-list randomly

Reply:
Did that

Reply:
👍